platform/linux-generic/odp_packet.c

```diff
@@ -1579,6 +1579,7 @@ void odp_packet_print(odp_packet_t pkt)
 	}
 	len += snprintf(&str[len], n - len, "  flags        0x%" PRIx32 "\n",
 			hdr->p.flags.all_flags);
+	len += snprintf(&str[len], n - len, "  cls_mark     %" PRIu16 "\n", hdr->cls_mark);
```

Comment:
Ok for this patch since it just follows the existing pattern. But the code is already broken because it does not take into account that the return value of snprintf is the number of characters that would have been written if the buffer were long enough and not the number actually written. But since that bug does not have anything to do with this patch, let's deal with it separately.

Reply:
Our snprintf() usage seems to rely on large enough str[] buffer size (4kB in this case). A macro/function could be used to wrap snprintf() return value check.